metadata-service/factories/src/main/java/com/linkedin/gms/factory/auth/AuthorizerChainFactory.java

```diff
@@ -65,53 +80,67 @@ protected AuthorizerChain getInstance() {
     return new AuthorizerChain(authorizers, dataHubAuthorizer);
   }
 
-  private AuthorizerContext initAuthorizerContext() {
-    final ResourceSpecResolver resolver = new DefaultResourceSpecResolver(systemAuthentication, entityClient);
-    return new AuthorizerContext(resolver);
+  private ResourceSpecResolver initResolver() {
+    return new DefaultResourceSpecResolver(systemAuthentication, entityClient);
   }
 
-  private List<Authorizer> initCustomAuthorizers(AuthorizerContext ctx) {
+  private List<Authorizer> initCustomAuthorizers(ResourceSpecResolver resolver) {
     final List<Authorizer> customAuthorizers = new ArrayList<>();
 
-    if (this.configurationProvider.getAuthorization().getAuthorizers() != null) {
-
-      final List<AuthorizerConfiguration> authorizerConfigurations =
-          this.configurationProvider.getAuthorization().getAuthorizers();
```

Comment:
Are we completely removing the ability to plugin authorizers here? What if people have already forked datahub and done that?

Reply:
they need to follow the plugin guide to accommodate the custom plugin and if they have any core plugin code then they can register it as we are doing for DataHubAuhorizer.